app/models/rubric_criterion.rb

```diff
@@ -101,19 +100,27 @@ def self.create_or_update_from_csv_row(row, assignment)
     if criterion.new_record?
       criterion.position = assignment.next_criterion_position
     end
-    # next comes the level names.
-    (0..RUBRIC_LEVELS - 1).each do |i|
-      criterion['level_' + i.to_s + '_name'] = working_row.shift
-    end
-    # the rest of the values are level descriptions.
-    (0..RUBRIC_LEVELS - 1).each do |i|
-      criterion['level_' + i.to_s + '_description'] = working_row.shift
-    end
-    unless criterion.save
-      raise CsvInvalidLineError
-    end
 
-    criterion
+    # create/update the levels
+    (0..RUBRIC_LEVELS - 1).each do
```

Comment:
@ashih2018 here, instead of using the constant `RUBRIC_LEVELS` (which we should remove altogether eventually), use the length of `working_row` to calculate how many levels there arel.

Reply:
for my next PR I will remove the constants and refactor the tests